minifi-nar-bundles/minifi-standard-nar/pom.xml

```diff
@@ -52,5 +52,60 @@ limitations under the License.
             <artifactId>nifi-standard-reporting-tasks</artifactId>
             <version>${org.apache.nifi.version}</version>
         </dependency>
+        <!-- Manage provided dependencies in lib -->
```

Comment:
If these are the deps and their versions that will be provided in lib then the version information should be set in the root pom. Also maybe even the scope but given the hiccups we've had on transitive "scoping" I'm hesitant.